src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+
+  --my-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.headerImg {
+  height: 40px;
+}
+
+.navigation {
+  width: 635px;
+  margin-left: auto;
+}
```

Comment:
you don't need margin-left auto and you shouldn't define width directly to navigations which is a container to links ([checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md): Don't fix container size (if there is no such a requirement))
instead add margins to items within :) also remember about requirements while making this changes: 
![image](https://github.com/user-attachments/assets/578c7869-a884-40bf-b5bd-9178b3afe01a)

Reply:
Dziękuje za tą wskazówkę, wszystkie testy zaliczone 🤩